example/App.tsx

```diff
@@ -7,7 +7,7 @@ import * as Maps from 'expo-maps';
 export default function App() {
   return (
     <View style={styles.container}>
-      <Maps.ExpoMap style={{ flex: 1, width: '100%' }} provider="apple" />
+      <Maps.ExpoMap style={{ flex: 1, width: '100%' }} mapType="satelite" />
```

Comment:
Shouldn't it be `satellite` instead of `satelite`? 😅 
```suggestion
      <Maps.ExpoMap style={{ flex: 1, width: '100%' }} mapType="satellite" />
```

Reply:
Yeah, it should 🤦